lib/watchdog/watchdog.go

```diff
@@ -0,0 +1,249 @@
+package watchdog
+
+import (
+	"context"
+	"errors"
+	"log"
+	"os"
+	"os/exec"
+	"os/signal"
+	"sync"
+	"sync/atomic"
+	"syscall"
+	"time"
+
+	"github.com/tarantool/tt/cli/process_utils"
+)
+
+type Watchdog struct {
```

Comment:
Please add here comment, too.
```suggestion
// Watchdog ......
type Watchdog struct {
```

Reply:
done